aspnet/migration/rc1-to-rc2.rst

```diff
@@ -180,36 +97,31 @@ WebApplicationConfiguration    WebHostConfiguration
 
 The ``commands`` section of ``project.json`` has been removed completely. Use ``dotnet run`` or ``dotnet <dllname>`` instead.
 
+Hosting Configuration 
+---------------------
+You enable configuration for your web application host by calling the ``UseConfiguration()`` method on ``WebHostBuilder`` and passing a configuration instance. The following configuration values can be specified:
 
-IHostingEnvironment changes 
----------------------------
+-application
+-startupAssembly
+-detailedErrors
+-server
+-webroot
+-captureStartupErrors
+-server.urls
+-contentRoot
 
-All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
+To configure hosting using environmental variables, you must add them as a configuration source, and optionally specify a prefix.
 
-===================================================  ===================================================
-RC1                                                  RC2
-===================================================  ===================================================
-ASPNET_APP, ASPNET_APPLICATIONNAME, Hosting:App      ASPNETCORE_APPLICATIONNAME
-ASPNET_STARTUPASSEMBLY                               ASPNETCORE_STARTUPASSEMBLY
-ASPNET_DETAILEDERRORS, Hosting:DetailedErrors        ASPNETCORE_DETAILEDERRORS
-ASPNET_ENVIRONMENT, ASPNET_ENV, Hosting:Environment  ASPNETCORE_ENVIRONMENT
-ASPNET_SERVER, Hosting:Server                        ASPNETCORE_SERVER
-ASPNET_WEBROOT, webroot                              ASPNETCORE_WEBROOT
-ASPNET_CAPTURESTARTUPERRORS                          ASPNETCORE_CAPTURESTARTUPERRORS
-ASPNET_SERVER.URLS                                   ASPNETCORE_SERVER.URLS
-ASPNET_CONTENTROOT, ASPNET_APPLICATIONBASE           ASPNETCORE_CONTENTROOT
-===================================================  ===================================================
-
-We still support ``ASPNET_ENV`` and ``Hosting:Environment`` in RC2, but the user will see a message indicating these values are deprecated. In RC2, you can use whatever prefix you want. You should add it explicitly by calling:
+ASPNETCORE_ENVIRONMENT is now how you specify the ASP.NET Core environment variable. We still support ``ASPNET_ENV`` and ``Hosting:Environment`` in RC2, but the user will see a message indicating these values are deprecated. In RC2, you can use whatever prefix you want. You should add it explicitly by calling:
 
 .. code-block:: c#
 
   new ConfigurationBuilder.AddEnvironmentVariables("ANY_PREFIX_YOU_WANT_").Build(); 
   
 However, there is an exception. You must set the environment key using ``ASPNETCORE_ENVIRONMENT``. This is picked up by default by the ``WebHostBuilder``, unlike the other variables. 
```

Comment:
> However, there is an exception. You must set the environment key using ``ASPNETCORE_ENVIRONMENT``. This is picked up by default by the ``WebHostBuilder``, unlike the other variables. 

However, there is an exception. You must set the environment key using ``ASPNETCORE_ENVIRONMENT``. This is picked up by default by the ``WebHostBuilder``, unlike the other variables. We still support ``ASPNET_ENV`` and ``Hosting:Environment`` in RC2, but the user will see a message indicating these values are deprecated.